integrated_channels/blackboard/migrations/0013_alter_blackboardlearnerdatatransmissionaudit_index_together.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 3.2.15 on 2022-12-20 15:05
+
+from django.db import connection, migrations
+
+
+class Migration(migrations.Migration):
+    atomic = False
+
+    dependencies = [
+        ('blackboard', '0012_move_and_recrete_completed_timestamp'),
+    ]
+
+    db_engine = connection.settings_dict['ENGINE']
+    if 'sqlite3' in db_engine:
+        operations = [
+            migrations.AlterIndexTogether(
+                name='blackboardlearnerdatatransmissionaudit',
+                index_together={('enterprise_customer_uuid', 'plugin_configuration_id')},
+            ),
+        ]
+    else:
+        operations = [
+            migrations.SeparateDatabaseAndState(
+                state_operations=[
+                    migrations.AlterIndexTogether(
+                        name='blackboardlearnerdatatransmissionaudit',
+                        index_together={('enterprise_customer_uuid', 'plugin_configuration_id')},
+                    ),
+                ],
+                database_operations=[
+                    migrations.RunSQL(sql="""
+                        CREATE INDEX "blackboard_blackboardlearnerdatatransmissionaudit_enterprise_customer_uuid_plugin_configuration_id_85936b55_idx"
+                        ON "blackboard_blackboardlearnerdatatransmissionaudit" ("enterprise_customer_uuid", "plugin_configuration_id")
+                        ALGORITHM=INPLACE LOCK=NONE
```

Comment:
the syntax `ALGORITHM=INPLACE LOCK=NONE` avoids a lock

Reply:
maybe add a link to some docs near this?

Reply:
will do, i am going to pop something into the wiki somewhere

Reply:
https://dev.mysql.com/doc/refman/5.7/en/innodb-online-ddl-operations.html